src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-type.mdx

```diff
@@ -93,32 +91,29 @@ Here are some tips for deciding on a user type:
 * They play a key role in the development, testing, deployment, and maintenance phases of the application development lifecycle.
 * They break/fix code regularly; they are responsible for triaging workflows, troubleshooting, or managing users and roles for their team.
 * They have DevOps practices (for example, version control systems, and implementation of CI/CD).
-* They need full access to our platform: They need to use New Relic's curated dashboards and experiences (not just the ability to create their own custom queries and charts).
-* They need to be able to manage users and/or billing.
-
+* They need full access to our platform: They need to use New Relic's curated dashboards and experiences (not just the ability to use custom-built queries and charts).
 
 **Reasons to make someone a [core user](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release):**
```

Comment:
This whole section is really good, btw. Learned a lot + will probably be referencing it to help me write to specific kinds of audiences.